mysql-test/suite/rocksdb/include/rpl_row_not_found.inc

```diff
@@ -10,7 +10,8 @@
 
 connection slave;
 call mtr.add_suppression("Slave SQL: Worker.*failed executing transaction");
```

Comment:
I think that is all it is, a 5.6 to 5.7 diff with 5.7 bringing in the "for channel" but can't say offhand for sure as I have not gone back and looked at the actual 5.6 error being logged.